tests/test_cart.py

```diff
@@ -361,3 +362,18 @@ def test_order_aggregate_vat_rounding(db):
 
     assert order.total() == 0
     assert order.total_vat_amount() == 0
+
+
+def test_order_aggregate_vat_rounding(db):
+    setup_conference_with_typical_fares()
+    vat = VatFactory(value=Decimal(20))
+
+    fare = FareFactory(price=100)
+    VatFareFactory(vat=vat, fare=fare)
+
+    order = OrderFactory(items=[
+        (fare, {"qty": 1}),
+    ])
+
+    assert order.total_vat_amount() < order.total()
+    assert order.total_vat_amount() == approx((order.total() * vat.value / 100) / (1 + vat.value / 100), abs=0.01)
```

Comment:
why approx here?

Reply:
Because of rounding, which was what issue #808 was all about.
Using VAT of 7.7%, if an item is listed as 99,- with VAT, it is calculated to 91,92 without VAT + 7.08 VAT
If you buy three of these items you get 297,- total. If you just calculate out the VAT from the total, you'd get 21,23 (as is done here), but that is actually wrong. It really is 21,24, because the per-item VAT is 7,08. It's probably possible to come up with cases where the difference is larger than 0,01 but that would need an order with more than one fare.

Reply:
I'll include a comment in the code, too - `order.total_vat_amount` is rounded, while the `approx`'ed calculation isn't.

Reply:
Ideally, we should move the rounding away from the model logic and into the presentation layer, as in - use a template filter. But that'd be a more significant change - I'll leave a comment to that effect in the model, too.